lib/types/PolicyRestrictions.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Kuzzle, a backend software, self-hostable and ready to use
+ * to power modern apps
+ *
+ * Copyright 2015-2022 Kuzzle
+ * mailto: support AT kuzzle.io
+ * website: http://kuzzle.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export type PolicyRestrictions = {
+  index: string,
+  collections: string[],
+};
+
+export type OptimizedPolicyRestrictions = Map<string, string[]>;
```

Comment:
Can you add a comment about what is used in the `string` and the `string[]` please?  
Like `Map<index, collection[]>`